stories/components/Radio/Radio.stories.tsx

```diff
@@ -291,16 +292,54 @@ const RadioMinimalStory: React.FC = () => {
         }}
       >
         <FormBox padding={2}>
+          <RadioGroupMinimal
```

Comment:
Most of these changes are just adding the two new icon examples and updating the old examples values and labels.